extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/ImporterPlugin.java

```diff
@@ -102,31 +97,18 @@ public static BuildServerConnection getBuildServerConnection(IPath rootPath, boo
             return null;
         }
 
-        String javaExecutablePath = getJavaExecutablePath();
-        String[] classpaths = getBuildServerClasspath();
-
-        String pluginPath = getBuildServerPluginPath();
-
-        List<String> command = new ArrayList<>();
-        command.add(javaExecutablePath);
-        if (Boolean.parseBoolean(System.getenv("DEBUG_GRADLE_BUILD_SERVER"))) {
-            command.add("-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=8989");
+        if (instance.buildServers.size() > 0) {
+            throw new CoreException(new Status(IStatus.ERROR, PLUGIN_ID,
+                    "Not support multiple workspaces."));
```

Comment:
logic here: should only have one buildServerconnection, one rootPath. if `instance.buildServers.get(rootPath) == null` and `buildServers.size()>0`, meaning that somehow errorly appear another rootPath.